lib/Parser/MimeDir.php

```diff
@@ -75,7 +91,7 @@ class MimeDir extends Parser
      *
      * @return \Sabre\VObject\Document
      */
-    public function parse($input = null, $options = 0)
+    public function parse($input = null, $options = 0, DateTimeInterface $start = null, DateTimeInterface $end = null)
```

Comment:
Do you really need the params here or is the public setTimeRange() method sufficient?

Reply:
Yeah I guess theres no real need for them. Think I left them in to keep it somewhat consistant with the freebusy generator. The call to setTimeRange can be removed from the parse method then too. 
I can do this tomorrow at some point.